engine/src/cmd/collide2/OPC_SweepAndPrune.cpp

```diff
@@ -201,9 +201,9 @@ SAP_Element* SAP_PairData::GetFreeElem(udword id, SAP_Element* next, udword* rem
 
 			// Remap everything
 			{
-				udword Delta = uintptr_t(NewElems) - uintptr_t(mElementPool);
+				ice_udword Delta = uintptr_t(NewElems) - uintptr_t(mElementPool);
```

Comment:
Hmm. Didn't catch this. `Delta` should probably be of type `ice_ptrdiff_t` or whatever it's called.

Reply:
probably should have a comment about the importance of the pointer arithmetic too

Reply:
Both done